README.md

```diff
@@ -36,14 +49,27 @@ To build the guide locally, take the following steps:
   # Build the book locally!
   $ jupyter-book build .
   ```
+* To view your built book:
 
-  To clean out old pages of your book:
+  Navigate to _build/html/ on your local clone of the repo and open "index.html".
+
+* To clean out old pages of your book (e.g. if you remove a page):
 
   ```
   $ jupyter-book clean .
   ```
 
-## Contributing to this guide
+## Automated build and publishing
+
+Whenever a pull request is opened or changes are pushed to any branch of the base repository, the book is built (separately) by both GitHub Actions and CircleCI. 
+
+### Why both GitHub Actions and CircleCI?
+
+- GitHub Actions is the main build tool. In addition to building whenever a pull request is opened from a fork, it handles publishing the book when changes are made to the main branch (e.g. once the pull request is merged). When that happens, it pushes the built html files to the gh-pages branch, which publishes the book to the [website](https://pyopensci.org/contributing-guide/).
+- CircleCI's build is redundant, but it offers an easier way of viewing the built html in browser WITHOUT merging the changes to main or downloading files. See [How do you preview the the guide from a Pull Request](https://github.com/pyopensci/contributing-guide/#how-do-you-preview-the-guide-from-a-pull-request) below for details. 
```

Comment:
I'm curious if this link works @kysolvik shouldn't it be `contributing-guide/README.md#how....`
or just the # anchor name? i haven't tried to use an anchor in a md file before like this so i could be wrong :)

Reply:
I thought the same, but when you get the link from the README it skips the README.md part and just puts the anchor directly after the repo name, e.g. https://github.com/pyopensci/contributing-guide/#build-the-guide-locally

Reply:
ok perfect!!